VARIABLES.md

```diff
@@ -406,20 +414,28 @@ Default:  "{{kafka_broker_configure_additional_brokers}}"
 
 ### kafka_broker_user
 
-Only use to customize Linux User Kafka Service runs with. User must exist on host.
+Only use to customize Linux User Kafka Service runs with. Default user is cp-kafka
 
 Default:  "{{kafka_broker_default_user}}"
 
 ***
 
 ### kafka_broker_group
 
-Only use to customize Linux Group Kafka Service user belongs to. Group must exist on host.
+Only use to customize Linux Group Kafka Service user belongs to. Default user is confluent
```

Comment:
Modify this variable to customize the Linux Group that the Kafka Service user belongs to. Default user is confluent.